framework/PageFramework.css

```diff
@@ -86,3 +86,7 @@ tr.selected:hover {
 }
 
 */
+
```

Comment:
We can use styled components here. In `SelectSingleDialog.tsx`:

```
const StyledModal = styled(Modal)`
  .pf-v5-c-modal-box__header {
    padding-bottom: 24px;
  }
`;
```

Reply:
The PF header does not have the right padding and then the content of the modal scrolls, it looks really bad.
The fix is to update it at the global level so if fixes all modals.